### PR TITLE
sensors added :D

### LINKED ISSUE #44: add sensors (subsystems)
- [x] one digital input laser sensor between acquisition and storage - should go into the acquisition subsystem.
- [x] one digital input laser sensor b/w storage and scoring loader - should go into the storage subsystem
- [x] one at top of scoring loader - should go into scoring

### PR DESCRIPTION
**Pull request by matunkar**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #44

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Acquisition.java [hunk 1]
```diff
@@ -28,6 +30,10 @@ public static Acquisition getInstance() {
         return instance;
     }
 
+    public boolean getLaserOutput() {
+        acqLaser.get();
+    }
+
```
Comment (Murat65536) [inline src/main/java/frc/robot/subsystems/Acquisition.java:36]: You are not returning anything. Add "void".

**Review by Murat65536 — CHANGES_REQUESTED** [review]

**Review by Griffin9881 — APPROVED** [review]
looks pretty adequate

File: src/main/java/frc/robot/subsystems/Acquisition.java [hunk 2]
```diff
@@ -28,6 +30,10 @@ public static Acquisition getInstance() {
         return instance;
     }
 
+    public boolean getLaserOutput() {
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Acquisition.java:33]:
```suggestion
    public boolean isNotePresent() {
```

File: src/main/java/frc/robot/subsystems/Scoring.java [hunk 3]
```diff
@@ -112,6 +114,10 @@ public void stopLoader() {
         leftLoadingMotor.stopMotor();
     }
 
+    public boolean getLaserOutput() {
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Scoring.java:117]:
```suggestion
    public boolean isNotePresent() {
```

File: src/main/java/frc/robot/subsystems/Storage.java [hunk 4]
```diff
@@ -27,7 +29,11 @@ private Storage() {
         storageMotor.setIdleMode(IdleMode.kBrake);
         storageMotor.burnFlash();
     }
-    
+
+    public boolean getLaserOutput() {
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Storage.java:33]:
```suggestion
    public boolean isNotePresent() {
```

File: src/main/java/frc/robot/subsystems/Scoring.java [hunk 5]
```diff
@@ -26,6 +27,7 @@ public class Scoring extends PIDSubsystem {
     private final CANSparkMax rightLoadingMotor = new CANSparkMax(
             ScoringConstants.rightScoringElevatorMotorPort, MotorType.kBrushless);
     private AbsoluteEncoder leftScoringElevatorEncoder = leftScoringElevatorMotor.getAbsoluteEncoder(Type.kDutyCycle);
+    private static DigitalInput loaderLaser = new DigitalInput(ScoringConstants.loaderLaserPort);
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Scoring.java:30]: This laser should be in the storage subsystem. I recommend calling them transitionLaser and storageSensor

**Review by reediculous456 — CHANGES_REQUESTED** [review]

**Comment by reediculous456** [comment]
Being completed by #64

**Pull request closed without merging**